src/containers/Storage/StorageNodes/columns/StorageNodesColumns.scss

```diff
@@ -9,6 +9,7 @@
         align-items: flex-end;
 
         width: max-content;
+        height: 40px;
```

Comment:
Make `PDisk` components to have the same height. It's needed to make `StorageNodes` to have the same row height, allowing switching from `variable` RDT rendering to `uniform`